documentation/src/main/asciidoc/chapter-5-data-type-conversions.asciidoc

```diff
@@ -253,6 +253,61 @@ This means that it is possible for MapStruct not to report unmapped target prope
 ====
 
 
+[[invoking-custom-mapping-method]]
+=== Invoking custom mapping method
+
+Sometimes mappings are not straightforward, and some fields should be mapped using plain java code, while mapping latter of them is covered by generated method.
+
+Let's see if there is a properties `length`, `width` and `height` in `FishTank`, and `VolumeDto` inner object, which contains `volume` and `description` fields in `FishTankWithVolumeDto`.
+We could convert rest of the fields using mappings defined inside `@Mapper` annotations, however it requires implementing method to create `VolumeDto`.
```

Comment:
defined inside `@Mapper` == defined by means of `@Mapping` annotations

Reply:
Fixed